[Issue] create project via CLI (Ubuntu 24)

### Details

### Details

### Problem

Application fails to start due to a missing native dependency: `@resvg/resvg-js-linux-x64-gnu`.

### Context

Occurs when running the backend/API service
The error originates from `@resvg/resvg-js` and propagates through `@hexabot-ai/api` services during app initialization.

### Steps to Reproduce (OS: Ubuntu 24)
1. `hexabot create project2`
2. `cd project2`
3. `hexabot start`
### Expected

Application should start successfully without missing module errors.

### Impact

Blocks local development and prevents the API service from starting on Ubuntu environments.

### Extra

Error:

```bash
Error: Cannot find module '@resvg/resvg-js-linux-x64-gnu'
code: 'MODULE_NOT_FOUND'
```

Stack trace points to:

```bash
node_modules/@resvg/resvg-js/js-binding.js
```

<img width="1280" height="677" alt="Image" src="https://github.com/user-attachments/assets/d438d024-5e6e-4d4a-a3be-0cf1482cd283" />

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -180,6 +180,7 @@
     "@css-inline/css-inline-linux-arm64-musl": "^0.14.1",
     "@resvg/resvg-js-darwin-arm64": "^2.6.2",
     "@resvg/resvg-js-linux-arm64-musl": "^2.6.2",
+    "@resvg/resvg-js-linux-x64-gnu": "2.6.2",
     "@resvg/resvg-js-win32-x64-msvc": "2.6.2"
   },
   "jest": {

